content/docs/command-reference/data/status.md

```diff
@@ -0,0 +1,134 @@
+# data status
+
+Show changes in the data tracked by DVC in the workspace.
+
+## Synopsis
+
+```usage
+usage: dvc data status [-h] [-q | -v]
+                       [--granular] [--unchanged]
+                       [--untracked-files [{no,all}]]
+                       [--json]
+```
+
+## Description
+
+The `data status` command displays the state of the working directory and the
+changes with respect to the last Git commit (`HEAD`).
+
+It shows you what new changes have been committed to DVC, which haven't been
```

Comment:
I really like the way it's written now. Simple and with a human in mind.

> you what's been going on with [dvc add](https://dvc-org-dvc-data-status-smqvo1.herokuapp.com/doc/command-reference/add), [dvc commit](https://dvc-org-dvc-data-status-smqvo1.herokuapp.com/doc/command-reference/commit) and [dvc repro](https://dvc-org-dvc-data-status-smqvo1.herokuapp.com/doc/command-reference/repro) commands

here it goes a bit too informal. I understand what you are trying to say, but I feel an urge to rephrase it a bit. Tbh we can drop it probably and combine the sentence left with the first intro paragraph.